docs/docsite/rst/using/installing.rst

```diff
@@ -18,7 +18,7 @@ For example, the following downloads the `debops.apt role <https://galaxy.ansibl
 
 .. code-block:: bash
 
-    $ ansible-galaxy install debops.apt
+    $ ansible-galaxy collection install debops.debops
```

Comment:
The topic and example are for installing a Role, not a collection. This change is inaccurate.